build.gradle

```diff
@@ -17,7 +17,7 @@
 apply plugin: 'java'
 apply from: 'plugin-common.gradle'
 
-project.ext.pluginVersion = '1.0.2'
+project.ext.pluginVersion = '2.0.0'
```

Comment:
Mind updating the GoCD version `goCdVersion` to `18.10.0`?

Reply:
Done